cni/network/network_windows.go

```diff
@@ -14,8 +14,8 @@ import (
 	"github.com/Azure/azure-container-networking/log"
 	"github.com/Azure/azure-container-networking/network"
 	"github.com/Azure/azure-container-networking/network/policy"
-	"github.com/Microsoft/hcsshim"
-	hnsv2 "github.com/Microsoft/hcsshim/hcn"
+	"github.com/microsoft/hcsshim"
```

Comment:
I think it is "M" not microsoft with lowercase. Lets correct that, once i made that change it helped in my local repo.